src/wrapped.rs

```diff
@@ -0,0 +1,112 @@
+#![allow(missing_docs)]
+
+use {StringInterner, Symbol, Iter, Values};
+use std::collections::hash_map::RandomState;
+use std::hash::BuildHasher;
+use std::ops::Deref;
+
+#[derive(Copy, Clone, Debug)]
```

Comment:
I think it should be possible to derive `Serialize` and `Deserialize` for serde support.

Reply:
I'm not entirely certain about this one (though if the derive works, it probably will). This is because of the reference/lifetime tying the ref back to the pool.